packages/core/strapi/lib/core/fs.js

```diff
@@ -44,6 +45,14 @@ module.exports = strapi => {
       const removePath = normalizePath(optPath);
       return fse.remove(removePath);
     },
+
+    /**
+     * Appends a file in strapi app
+     */
+    async appendFile(optPath, data) {
+      const writePath = normalizePath(optPath);
+      return fs.appendFileSync(writePath, data);
```

Comment:
you can use fse.appendFile instead to keep using fs-extra

Reply:
I don't find it in their doc : https://github.com/jprichardson/node-fs-extra

Reply:
that's just because it's not documented but they export all the same function as fs so they didn't bother document as they have the same API just promisified: https://github.com/jprichardson/node-fs-extra/blob/master/lib/fs/index.js#L9